pyqt-apps/siriushla/si_ap_fofb/custom_widgets.py

```diff
@@ -348,7 +348,7 @@ def _setupUi(self):
                 glay2.addWidget(led, 1, 2)
 
                 lbl = QLabel(
-                    'Orbit Distortion Threshold: ', self,
+                    'Orbit Distortion Threshold (um): ', self,
```

Comment:
try to use `[]` instead of `()` for units.

Reply:
done!